librz/include/rz_analysis.h

```diff
@@ -523,6 +524,7 @@ typedef struct rz_analysis_options_t {
 	bool trycatch;
 	bool norevisit;
 	int afterjmp; // continue analysis after jmp eax or forward jmp // option
+	int aftertrap; // continue analysis after trap instructions
```

Comment:
This might be useful for many architectures, indeed.

Reply:
Does also works for all architectures. But the config is turned of by default.